clientapi/routing/profile.go

```diff
@@ -300,7 +300,7 @@ func updateProfile(
 		}, e
 	}
 
-	if err := api.SendEvents(ctx, rsAPI, api.KindNew, events, device.UserDomain(), domain, domain, nil, true); err != nil {
+	if err := api.SendEvents(ctx, rsAPI, api.KindNew, events, device.UserDomain(), domain, domain, nil, false); err != nil {
```

Comment:
This slows down changing profiles, but makes sure that all events have been sent to the roomserver.
This is needed, as e.g. bridges may change the avatar and displayname in short succession.